flycheck.el

```diff
@@ -6241,6 +6242,30 @@ See URL `https://github.com/kisielk/errcheck'."
     ;; packages, and can't check individual Go files.
     (and (flycheck-buffer-saved-p) (flycheck-go-package-name))))
 
+(flycheck-define-checker groovy
+  "A groovy syntax checker using groovy compiler API.
+
+See `http://www.groovy-lang.org/mailing-lists.html#nabble-td365810'
+and `http://docs.groovy-lang.org/latest/html/gapi/org/codehaus/groovy/control/CompilationUnit.html#compile%28int%29'."
+
+  :command ("groovy" "-e"
+            (eval "import org.codehaus.groovy.control.*
```

Comment:
You don't need `eval` here.  It's just a literal string, which you can pass without any special form.